org.eclipse.wildwebdeveloper/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Wild Web Developer: web development in Eclipse IDE
 Bundle-SymbolicName: org.eclipse.wildwebdeveloper;singleton:=true
 Automatic-Module-Name: org.eclipse.wildwebdeveloper
-Bundle-Version: 0.2.0.qualifier
+Bundle-Version: 0.2.1.qualifier
```

Comment:
Since it's a relatively noticeable addition, we'll bump to 0.3.0 to showcase there is more than maintenance fix.